src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -22,13 +22,11 @@ public sealed class ConfigurationFeatureDefinitionProvider : IFeatureDefinitionP
         // IFeatureDefinitionProviderCacheable interface is only used to mark this provider as cacheable. This allows our test suite's
         // provider to be marked for caching as well.
 
-        private const string FeatureFiltersSectionName = "EnabledFor";
-        private const string RequirementTypeKeyword = "RequirementType";
-        private const string FeatureManagementSectionName = "FeatureManagement";
         private readonly IConfiguration _configuration;
         private readonly ConcurrentDictionary<string, FeatureDefinition> _definitions;
         private IDisposable _changeSubscription;
         private int _stale = 0;
+        private bool _serverSideSchemaEnabled;
```

Comment:
Is `_isFeatureFlagArraySchema` better? Anyway，we'd avoid using the “serverSide” word, since it is not meaningful for FeatureManagement library